src/main/java/com/softech/coursesvc/CourseRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.softech.coursesvc;
+
+import com.softech.coursesvc.model.Course;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+
+public interface CourseRepository extends JpaRepository<Course, Long>{
+
+    Course findById(long id);
```

Comment:
You dont need to add this method manully..this method is automatically provided by default